localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/README.md

```diff
@@ -0,0 +1,61 @@
+# ndt_scan_matcher
+
+## Purpose
+
+ndt_scan_matcher is a package for position estimation using the NDT scan matching method.
+
+There are two main functions of this package:
```

Comment:
Maybe `There are two main functions in this package`?

Reply:
Thank you! https://github.com/tier4/AutowareArchitectureProposal.iv/pull/690/commits/bb61812d80b259692c01e35b2d082c315696ef5f